cmd/rollback.go

```diff
@@ -81,6 +82,7 @@ func rollbackCmd() *cobra.Command {
 	rollbackCmd.Flags().BoolVar(&diff.detailedExitCode, "detailed-exitcode", false, "return a non-zero exit code when there are changes")
 	rollbackCmd.Flags().StringArrayVar(&diff.suppressedKinds, "suppress", []string{}, "allows suppression of the values listed in the diff output")
 	rollbackCmd.Flags().IntVarP(&diff.outputContext, "context", "C", -1, "output NUM lines of context around changes")
+	rollbackCmd.Flags().Float32VarP(&diff.maxChangeFraction, "max-content-change-fraction", "D", 0, "maximum fraction of changed content as lines added + removed compared to total lines in a diff for considering it a moved resource")
```

Comment:
```suggestion
	rollbackCmd.Flags().Float32VarP(&diff.findRenames, "find-renames", "D", 0, "Enable rename detection if set to any value greater than 0. If specified, the value denotes the maximum fraction of changed content as lines added + removed compared to total lines in a diff for considering it a rename")
```

How about using `find-renames` like the similar feature exists in `git diff`?

```
       -M[<n>], --find-renames[=<n>]
           Detect renames. If n is specified, it is a threshold on the similarity index (i.e. amount of addition/deletions compared to the file's size). For example, -M90% means Git should consider a delete/add pair
           to be a rename if more than 90% of the file hasn't changed. Without a % sign, the number is to be read as a fraction, with a decimal point before it. I.e., -M5 becomes 0.5, and is thus the same as -M50%.
           Similarly, -M05 is the same as -M5%. To limit detection to exact renames, use -M100%. The default similarity index is 50%.
```

Reply:
It is indeed inspired by `find-renames`... 

 I was doubting if the expectation of a `find-renames` would be to only discover changes of `metadata.name` and possibly `metadata.namespace`, but not the kind. 
My problem though is, that the proposal also detects changes of the `Kind` (e.g. `Deployment` -> `StatefulSet`).

But since your intuition is that it is the same feature, I may be overthinking things here.

In that light, what is your opinion? If you still say it is more intuitive to use the same parameters, I will change it. I wasn't sure myself.

Reply:
> My problem though is, that the proposal also detects changes of the Kind (e.g. Deployment -> StatefulSet).

Ah I missed this part. To me, it seems like it might benefit from having another flag to detect kind change, like `--find-kind-changes`? But not sure if that harmonizes with `--find-renames` then...

BTW, can we just omit the kind change detection? In most cases a kind change would result in a huge diff in spec and I'd say changing from a deployment to statefulset or vice versa doesn't happen so often. Rename detections sounds useful in more cases than kind change detection and complicating it for less used feature doesn't seem to pay off

Reply:
Sure, I will restrict it. That also reduces also the search-space. I just thought this is more simple to implement and covers more use-cases.